test/integration/integration-cli-help.test.ts

```diff
@@ -15,7 +15,7 @@ async function prepareTestProjectWithProbot() {
     "..",
   );
 
-  await execCommand("npm link", {
+  await execCommand("npm link --force", {
```

Comment:
maybe add a comment explaining the `--force` for future reference?

Reply:
I am actually confused, that deno is still erroring.

Reply:
trying a different approach now